lib/ui/bloc/chat_cubit.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:fv_chat/domain/entities/chat_message.dart';
+import 'package:fv_chat/data/repository/ai_repository_impl.dart';
+import 'package:fv_chat/di/di.dart';
+
+class ChatState {
```

Comment:
move to separate file.

use @freezed lib with state it will create copyWith method under the hood